examples/sorting_data.py

```diff
@@ -0,0 +1,85 @@
+"""
+=========================
+Filtering and Visualizing
+=========================
+
+This example provides a simple overview of filtering and visualizing XRT data.
+"""
+
+import astropy.units as u
+import matplotlib.pyplot as plt
+import sunpy.map
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+##############################################################################
+# To start we will download a range of XRT data from the Virtual Solar Observatory (VSO).
+# The goal is to acquire a large set of files we can sort through and visualize.
+
+query = Fido.search(
+    a.Time("2021-05-21 18:51:00", "2021-05-22 00:00:00"), a.Instrument("xrt")
+)
+print(query)
+
+##############################################################################
+# This query will return a large number of files, this is due to the fact we do not
+# specify any additional filters. We can filter the data by specifying additional
+# attributes in the query.
+#
+# For wavelength, we use a range that focuses the data to return only Al-Poly filter images.
+# This will cut the results down in half.
+
+query = Fido.search(
+    a.Time("2021-05-21 20:51:00", "2021-05-22 00:00:00"),
+    a.Instrument("xrt"),
+    a.Wavelength(4 * u.nm, 5 * u.nm),
+)
+print(query)
+
+##############################################################################
+# Now we will download the data.
+# As this is still over 60 files, this process can take some time.
+
+xrt_files = Fido.fetch(query)
+
+##############################################################################
+# We can now load the data into a `~sunpy.map.MapSequence` and create a animation.
+
+xrt_seq = sunpy.map.Map(xrt_files, sequence=True)
+
+fig = plt.figure()
+ax = fig.add_subplot(projection=xrt_seq.maps[0])
+ani = xrt_seq.plot(axes=ax)
```

Comment:
For all of these animations, the normalization should be specified (e.g. https://docs.sunpy.org/en/stable/generated/gallery/map/difference_images.html#sphx-glr-generated-gallery-map-difference-images-py) explicitly. Otherwise, it changes from frame to frame which makes it near impossible to actually understand what is being animated.

Reply:
Should be fixed now with the new norm.